### PR TITLE
Update LastPass.xml

### PR DESCRIPTION
**Pull request by DeathlyX**

### CODE REVIEW

File: src/chrome/content/rules/LastPass.xml [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
   <target host="lastpass.com" />
   <target host="*.lastpass.com" />
 
   <securecookie host="^(?:.+\.)?lastpass\.com$" name=".*"/>
 
-  <rule from="^http://(?:www\.)?lastpass\.com/" to="https://lastpass.com/"/>
-
-	<rule from="^http://forums\.lastpass\.com/"
-		to="https://forums.lastpass.com/" />
+  <rule from="^http:" to="https:" />
```
Comment (jsha) [inline src/chrome/content/rules/LastPass.xml]: The old version of this rule rewrote only forums and www, but the new version will rewrite all subdomains. I suggest you should probably keep the old semantics unless you have a very strong reason not to.

That means changing the target host = "*.lastpass.com" to target host = "forums.lastpass.com", target host = "www.lastpass.com".

Comment (DeathlyX) [inline src/chrome/content/rules/LastPass.xml]: I've done that deliberately. All the other sub-domains worked too apparently. The certificate name of forum.lastpass.com for instance says it applies to *.lastpass.com.

Comment (jsha) [inline src/chrome/content/rules/LastPass.xml]: Which other subdomains have you tested? You should test at least three, preferably more (you can often find subdomains using Wolfram Alpha). You should also add those subdomains as `<test url=... />`.

Comment (DeathlyX) [inline src/chrome/content/rules/LastPass.xml]: I've actually used Wolfram Alpha to find the sub-domains. I've tested all the sub-domains which were listed and they all worked. One sub-domain, download-do.lastpass.com, gave a 404, but it has nothing to do with http or https.

Comment (DeathlyX) [inline src/chrome/content/rules/LastPass.xml]: I've added some <test url=... /> lines.

**Comment by DeathlyX** [comment]
I can remove the wildcard if really necessary, but I believe it's justified. Should I change the ruleset?

**Pull request merged**